op-node/rollup/driver/state.go

```diff
@@ -290,7 +290,7 @@ func (s *Driver) eventLoop() {
 			s.log.Debug("Derivation process step", "onto_origin", s.derivation.Origin(), "attempts", stepAttempts)
 			err := s.derivation.Step(context.Background())
 			stepAttempts += 1 // count as attempt by default. We reset to 0 if we are making healthy progress.
-			if err == io.EOF {
+			if err == io.EOF || (err != nil && errors.Is(err, derive.EngineP2PSyncing)) {
 				s.log.Debug("Derivation process went idle", "progress", s.derivation.Origin())
```

Comment:
If `errors.Is(err, derive.EngineP2PSyncing)` is true, can we add more log messages to include error messages;
```suggestion
				s.log.Debug("Derivation process went idle", "progress", s.derivation.Origin(), "err", err)
```